src/schema-generation/filter-input-types/constants.ts

```diff
@@ -72,7 +77,14 @@ export const FILTER_DESCRIPTIONS: { [name: string]: string | { [typeName: string
     [INPUT_FIELD_NOT_LIKE]: 'Checks if $field does *not* match a pattern case-insensitively with the following placeholders:\n\n' +
     '- `%` matches any sequence of characters, including the empty string\n' +
     '- `_` matches exactly one character\n' +
-    '- `\\` can be used to escape the placeholders (use `\\\\` for a literal backslash)'
+    '- `\\` can be used to escape the placeholders (use `\\\\` for a literal backslash)',
+
+    [INPUT_FIELD_IN]: 'Checks if $field is contained in a specified list.',
```

Comment:
Maybe *Checks if $field is equal to one of the specified values* would be clearer